### ISSUE TITLE
Scope

### ISSUE THREAD

**Issue by dhruvdhody** [elear/rabbit#10]
>    This policy applies to all IETF plenary lists mentioned in [RFC9245].
>    In addition, certain aspects of this policy may be applied across all
>    IETF lists, as stated below.

I am unsure of which list are in scope? RFC 9245 mentions architecture-discuss which is an IAB list. The second sentence all IETF lists is also unclear... I suggest we should make it explicitly clear - which list are included.

**Comment by jrlevine** [comment]
RFC 9245 is about ietf@ietf.org. I suppose the sentence quoted about should say "list" rather than "lists:.

**Comment by dhruvdhody** [comment]
I agree but when I saw "all" and "lists" in the quoted text, I assumed that was not what was intended... 

Also note that BCP83 which you are replacing says that --> 

> (For the purposes of this memo, the term "IETF mailing
>    list" refers to any mailing list functioning under IETF auspices,
>    such as the IETF general discussion list, or a working group or
>    design team mailing list.)

So just ietf@ietf.org won't cut it...

I also feel we need to explicitly say IAB, IRTF, and RFC-EDITORs lists are in or out of scope...
